extensions/amp-story/1.0/amp-story.js

```diff
@@ -438,6 +444,10 @@ export class AmpStory extends AMP.BaseElement {
       this.element.removeChild(node);
     });
 
+    if (this.viewerMessagingHandler_) {
+      this.viewerMessagingHandler_.startListening();
```

Comment:
Curious, why do this here instead of `initializeListeners_()`?

Reply:
Yep good point, done.